ui/components/app/assets/asset-list/asset-list-control-bar/use-account-total-fiat-balances.ts

```diff
@@ -0,0 +1,41 @@
+import { shallowEqual, useSelector } from 'react-redux';
+import { useAccountTotalFiatBalance } from '../../../../../hooks/useAccountTotalFiatBalance';
+import {
+  getConfirmationExchangeRates,
+  getSelectedAccount,
+  getShouldHideZeroBalanceTokens,
+  getTokenExchangeRates,
+} from '../../../../../selectors';
+
+export const useAccountTotalFiatBalancesHook = () => {
+  // Selectors from the Redux store
+  const selectedAccount = useSelector(getSelectedAccount);
+  const shouldHideZeroBalanceTokens = useSelector(
+    getShouldHideZeroBalanceTokens,
+  );
+
+  // Fetching exchange rates from the Redux store
+  const contractExchangeRates = useSelector(
+    getTokenExchangeRates,
+    shallowEqual,
+  );
+  const confirmationExchangeRates = useSelector(getConfirmationExchangeRates);
+
+  // Merging exchange rates
+  const mergedRates = {
+    ...contractExchangeRates,
+    ...confirmationExchangeRates,
+  };
+
+  // Getting total fiat balance for the account
+  const accountTotalFiatBalance = useAccountTotalFiatBalance(
+    selectedAccount,
+    shouldHideZeroBalanceTokens,
+  );
+
+  return {
+    accountTotalFiatBalance,
+    mergedRates,
+    loading: accountTotalFiatBalance.loading,
+  };
+};
```

Comment:
I think I am going to consolidate this logic into the existing `useAccountTotalFiatBalance` hook. It probably should have been this way to begin with.

Reply:
https://github.com/MetaMask/metamask-extension/pull/27184/commits/dccebb324786ed3d0cd11e063d78fa856ffe1b67 and https://github.com/MetaMask/metamask-extension/pull/27184/commits/86c43a9cdd6f72bf7d557881329c2c32f609e0e5